index.html

```diff
@@ -14017,49 +14026,42 @@ <h4 id="PrimaryGlobal">4.3.6. [Global] and [PrimaryGlobal]</h4>
                   <var>idlarg</var><sub><var>i</var></sub> to an ECMAScript value.
                 </li>
-                <li>Let <var>script</var> be the <a class="dfnref" href="#dfn-callback-context">callback context</a> associated with <var>V</var>.</li>
-                <li>Push <var>script</var> on to the <a class="dfnref external" href="http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html">stack of incumbent scripts</a>. <a href="#ref-HTML">[HTML]</a></li>
+                <li>Let <var>execution context</var> be <a class="external" href="https://tc39.github.io/ecma262/#sec-getfunctionrealm">GetFunctionRealm</a>(<var>X</var>)'s <a class="dfnref external" href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-settings-object">settings object</a>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#realm-execution-context">realm execution context</a>.</li>
+                <li>Increment <var>execution context</var>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#entrance-counter">entrance counter</a> by one.</li>
+                <li>Push <var>execution context</var> onto the <a class="external" href="https://tc39.github.io/ecma262/#execution-context-stack">JavaScript execution context stack</a>; it is now the <a class="external" href="https://tc39.github.io/ecma262/#running-execution-context">running JavaScript execution context</a>.</li>
                 <li>Let <var>R</var> be an uninitialized variable.</li>
                 <li>Try running the following step:
                   <ol>
-                    <li>Set <var>R</var> to the result of invoking the <span class="prop">[[Call]]</span> method of <var>X</var>, providing <var>this</var> as the <span class="esvalue">this</span> value and <var>arg</var><sub>0..<var>n</var>−1</sub> as the argument values.</li>
+                    <li>Set <var>R</var> to <a class="external" href="https://tc39.github.io/ecma262/#sec-call">Call</a>(<var>X</var>, <var>thisArg</var>, <var>arg</var><sub>0..<var>n</var>−1</sub>).</li>
                   </ol>
                   And then, whether or not an exception was thrown:
                   <ol>
-                    <li>Pop <var>script</var> off the <a class="dfnref external" href="http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html">stack of incumbent scripts</a>.</li>
+                    <li>Decrement <var>execution context</var>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#entrance-counter">entrance counter</a> by one.</li>
+                    <li>Remove <var>execution context</var> from the <a class="external" href="https://tc39.github.io/ecma262/#execution-context-stack">JavaScript execution context stack</a>.</li>
                     <li>If an exception was thrown, end these steps, and allow it to propagate.</li>
```

Comment:
I'm not sure that works right, though it's preexisting; this goes back to where most invokers want to report the exception if there is one, but the "where to report it" location is ... what exactly?  Once we have propagated out of here, we've taken down our entry settings, which iirc is the place you're supposed to report exceptions.